editor/timeline_displayer.gd

```diff
@@ -3,6 +3,9 @@ extends Tree
 
 const TimelineClass = preload("res://addons/blockflow/timeline.gd")
 const FALLBACK_ICON = preload("res://icon.svg")
+const BOOKMARK_ICON = preload("res://addons/blockflow/icons/bookmark.svg")
+const STOP_ICON = preload("res://addons/blockflow/icons/Stop.svg")
+const CONTINUE_ICON = preload("res://addons/blockflow/icons/Play.svg")
```

Comment:
change `Play` to `play`, we try to keep all names in `snake_case` to avoid problems on exportation on machines that distinguish between uppercase and lowercase characters